auto-value-gson/src/main/java/com/ryanharter/auto/value/gson/AutoValueGsonExtension.java

```diff
@@ -358,7 +358,7 @@ private static AnnotationSpec createGeneratedAnnotationSpec(TypeElement generate
     Set<TypeName> seenTypes = Sets.newHashSet();
     NameAllocator nameAllocator = new NameAllocator();
     for (Property property : properties) {
-      if (!property.shouldDeserialize() && !property.shouldSerialize()) {
+      if (property.shouldNotDeserialize() && !property.shouldNotSerialize()) {
```

Comment:
Should this be `property.shouldNotSerialize()` (drop the `!`)?

Reply:
Derp, will fix

Reply:
2dd551a